engine/src/flutter/lib/ui/text.dart

```diff
@@ -53,42 +55,47 @@ enum FontStyle {
 /// with the name "Roboto" and the weight [FontWeight.w500].
 ///
 /// Some modern fonts allow the weight to be adjusted in arbitrary increments.
-/// See [FontVariation.weight] for details.
+/// When using these fonts, applications can specify [FontWeight] instances
+/// constructed using values other than the predefined values.
```

Comment:
nit: maybe cross ref `FontVariation.weight` and how it interacts with `FontWeight`?